ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties

```diff
@@ -17,7 +17,7 @@ h2.db.port=8082
 server.port=8080
 server.address=localhost
 server.servlet.context-path=/${artifactId}
-server.tomcat.additional-tld-skip-patterns=xercesImpl.jar,xml-apis.jar,serializer.jar
+server.tomcat.additional-tld-skip-patterns=*.jar
```

Comment:
usage of *jar can cause issues with loading blue consel

Reply:
fixed